configure.ac

```
@@ -294,18 +294,18 @@ if test x"$va_copy" != xva_copy ; then
   )
 fi
 
-dnl Verify people aren't doing stupid shit
+dnl Avoid footguns.
 if test x"$enable_static" != xno ; then
-  AC_MSG_ERROR([dont be a Kumba, building a libsandbox.a is stupid])
+  AC_MSG_ERROR([Building a libsandbox.a is stupid; aborting])
```

If we are aiming to be less offensive, let's change this to something like "is not supported".
```suggestion
  AC_MSG_ERROR([Building a static libsandbox.a is not supported])
```

---

Thanks. I should note that it was both about being a bit less offensive, but also about not confusing people with WTF a "Kumba" is.